server/src/routines/game.ts

```diff
@@ -668,14 +671,9 @@ export function* turnSaga(game: GameModel) {
 	}
 	game.state.modalRequests = []
 
-	let deadPlayers: Array<PlayerComponent> = []
-	deadPlayers.push(...(yield* call(checkHermitHealth, game)))
-	deadPlayers.push(...(yield* call(checkDeckedOut, game)))
-
+	const deadPlayers: PlayerComponent[] = yield* call(checkHermitHealth, game)
 	if (deadPlayers.length) {
-		if (deadPlayers[0].deckedOut) {
-			game.endInfo.reason = 'cards'
-		} else if (deadPlayers[0].lives <= 0) {
+		if (deadPlayers.every((player) => player.lives <= 0)) {
```

Comment:
Shouldn't this be `deadPlayers.some()`? Also the same thing for the other occurrence.

Reply:
If the game ends in a tie where one player is out of lives and the other is out of hermits, should they both see the message "You ran out of `lives`" or "You ran of `hermits`"? (current patch shows whatever `playerOne` ran out of)

Reply:
Ok I see! It doesn't really matter to me so I like this implementation.